### PR TITLE
changes for pcm-1018, initial pass

### PR DESCRIPTION
**Pull request by pbathia**

### CODE REVIEW

**Comment by pbathia** [comment]
@ardasgup @sshumake Please review and merge

File: karma.conf.js [hunk 1]
```diff
@@ -102,7 +104,7 @@ module.exports = function(config) {
         // - Safari (only Mac)
         // - PhantomJS
         // - IE (only Windows)
-        browsers: ['PhantomJS'],
+        browsers: ['Chrome'],
```
Comment (sshumake) [inline karma.conf.js]: @pbathia  This should be changed back to PhantomJS

**Comment by pbathia** [comment]
@sshumake Changed back to PhantonJS

**Comment by sshumake** [comment]
@pbathia  looks good

**Comment by pbathia** [comment]
@ardasgup may be yes..but if you see mockStrataService.js file, it has many services defined not even related to strata like MockProductsService,MockDiscussionService, etc.

**Comment by pbathia** [comment]
@i have moved UDS mock stuff to other file. Have a loot at it.

**Comment by ardasgup** [comment]
Reviewed!

**Pull request merged**